Source/monster.cpp

```diff
@@ -50,6 +50,19 @@ void NewMonsterAnim(int i, AnimStruct *anim, Direction md, AnimationDistribution
 	monst->_mdir = md;
 }
 
+void StartMonsterGotHit(int mid)

```

Comment:
Can we take the opportunity and actually pass a `MonstStruct` reference here instead of the id?

Reply:
Currently that's not possible cause `M_ClearSquares` and `dMonster` need the index.